.github/workflows/test.yml

```diff
@@ -175,6 +175,44 @@ jobs:
         $env:_LIMA_WINDOWS_EXTRA_PATH = 'C:\Program Files\Git\usr\bin'
         bash.exe -c "./hack/test-templates.sh templates/experimental/wsl2.yaml"
 
+  windows-qemu:
```

Comment:
Can we now drop these lines?

https://github.com/lima-vm/lima/blob/df13f3003192f8f1a82944edd6296f99a1e125b8/pkg/limayaml/defaults.go#L68-L75

Reply:
From my experience "max" just didn't work well with WHPX acceleration. I tested it on 3 different machines in the past. I only was able to make it work by disabling specific CPU features, which were different on every machine. It was not user friendly default. I can do some canary testing to compare how it works now with newer QEMU/Windows versions and if the failures are as common as they were before.